### ISSUE TITLE
Create a CTA button that is a link-to

### ISSUE THREAD

**Issue by jenweber** [cardstack/ui-components#155]
In card builder, I found myself doing things like this:

```
<LinkTo
  @route="cards.card.edit.fields"
   @class="right-edge--configure-schema-link cs-component-cta secondary responsive cs-dark-cta" >
```

I stole the classes from the CTA button. We should create a CTA button that is a link-to instead, and replace any Link-Tos that have a class "cs-component-cta."

**Comment by jenweber** [comment]
@burieberry do you have any thoughts on whether we should do this and what the approach should be?

**Comment by burieberry** [comment]
@jenweber Yes, I think it's a good idea. I'm open to whichever approach seems easy and sturdy. To my horror, I saw a CTA component that's wrapping a link-to in Cardhost, and we gotta get rid of that.

**Comment by jenweber** [comment]
@burieberry yeah, that causes problems. I fixed one of those here <https://github.com/cardstack/cardstack/pull/1309/files>
